docs/manual-CN/3.build-develop-and-administration/2.install/1.install-with-rpm-deb.md

```diff
@@ -21,34 +21,34 @@
 
     1. 获取 release 版本，URL 格式如下：
 
-     * [Centos 6](https://nebula-graph.oss-cn-hangzhou.aliyuncs.com/package/${release_version}/nebula-${release_version}.el6-5.x86_64.rpm)
+     * [Centos 6](https://oss-cdn.nebula-graph.com.cn/package/${release_version}/nebula-${release_version}.el6-5.x86_64.rpm)
 
-     * [Centos 7](https://nebula-graph.oss-cn-hangzhou.aliyuncs.com/package/${release_version}/nebula-${release_version}.el7-5.x86_64.rpm)
+     * [Centos 7](https://oss-cdn.nebula-graph.com.cn/package/${release_version}/nebula-${release_version}.el7-5.x86_64.rpm)
 
-     * [Ubuntu 1604](https://nebula-graph.oss-cn-hangzhou.aliyuncs.com/package/${release_version}/nebula-${release_version}.ubuntu1604.amd64.deb)
+     * [Ubuntu 1604](https://oss-cdn.nebula-graph.com.cn/package/${release_version}/nebula-${release_version}.ubuntu1604.amd64.deb)
 
-     * [Ubuntu 1604](https://nebula-graph.oss-cn-hangzhou.aliyuncs.com/package/${release_version}/nebula-${release_version}.ubuntu1804.amd64.deb)
+     * [Ubuntu 1604](https://oss-cdn.nebula-graph.com.cn/package/${release_version}/nebula-${release_version}.ubuntu1804.amd64.deb)
```

Comment:
Ubuntu 1804， btw,  it returns an error when users click it

Reply:
the example is below